go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.65.0 // indirect
+	github.com/bazelbuild/bazelisk v1.8.0 // indirect
```

Comment:
Bazelisk is a simple wrapper to ensure users have the same _bazel_ version. While having the same `bazelisk` version might be useful, that shouldn't affect build performance/correctness/determinism. Usually there's a `.bazelrc` at the workspace's root specifying the .bazelversion that apparently we haven't bothered including on the SDK.

Bazelisk isn't an actual build dependency, so we shouldn't be adding it to `go.mod` :)

Reply:
got it. removed. ptal